Revert "openaptx: Blacklist >= 0.2.1 due to license change"

This reverts commit d08b6fac6bec0d334ee9fc785d551a67832f95fe.

Doesn't apply to us.

Bug: https://bugs.gentoo.org/785634

--- a/spa/meson.build
+++ b/spa/meson.build
@@ -22,7 +22,7 @@ if not get_option('spa-plugins').disabled()
   sbc_dep = dependency('sbc', required: get_option('bluez5'))
   ldac_dep = dependency('ldacBT-enc', required : get_option('bluez5-codec-ldac'))
   ldac_abr_dep = dependency('ldacBT-abr', required : get_option('bluez5-codec-ldac'))
-  aptx_dep = dependency('libopenaptx', version : '< 0.2.1', required : get_option('bluez5-codec-aptx'))
+  aptx_dep = dependency('libopenaptx', required : get_option('bluez5-codec-aptx'))
   fdk_aac_dep = dependency('fdk-aac', required : get_option('bluez5-codec-aac'))
   avcodec_dep = dependency('libavcodec', required: get_option('ffmpeg'))
   jack_dep = dependency('jack', version : '>= 1.9.10', required: get_option('jack'))
 
